client_lib/pulp/client/commands/polling.py

```diff
@@ -296,6 +296,19 @@ def succeeded(self, task):
         msg = _('Task Succeeded')
         self.prompt.render_success_message(msg, tag='succeeded')
 
+    def _render_coded_error(self, e):
+        """
+        Recursively render a coded error and the errors contained by it.
+
+        :param e: error to display
+        :type  e: dictionary containing error information.
+        """
+        malformed_msg = _('Request error does not contain a description, '
```

Comment:
Replace the comma with a semicolon, or make it a period and start a new sentence on the next line.